content/en/notebooks/advanced_analysis/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Analysis Features
+further_reading:
+- link: "/notebooks"
+  tag: "Documentation"
+  text: "Learn more about Notebooks"
+---
+{{< callout url="#" header="Join the Preview!">}} 
+  Advanced Analysis is in Preview
+{{< /callout >}}
+
+## Overview
+
+The computational functionality within Notebooks supports complex operations, such as combining attributes from multiple sources or transforming data, while still providing all the powerful features that Notebooks offer. 
+
+Notebooks are collaborative text editors that let you embed Datadog graphs directly into your documents. While this is great for exploration and storytelling, investigations may require more advanced control over data queries. Use analysis features to run queries that help you:
+
+* Correlate multiple data sources  
+* Aggregate multiple levels of data  
```

Comment:
Replace by "Chain queries: for example aggregate already aggregated data or join two sets of aggregated data